apps/smart-contracts/core/test/PrePOMarket.test.ts

```diff
@@ -556,24 +648,69 @@ describe('=> prePOMarket', () => {
       expect(await collateralToken.balanceOf(user.address)).to.eq(totalOwed.sub(redeemFee))
     })
 
-    it('should not allow redemption amounts too small for a fee before market end', async () => {
+    it('allows a redemption > 0 if redemption fee = 0% and redeeming equal parts', async () => {
```

Comment:
```suggestion
    it('allows amounts > 0 if redemption fee = 0% and redeeming equal parts', async () => {
```
nit: no `a`
same for below